e2e/windows/e2e-helper.sh

```diff
@@ -16,11 +16,23 @@ exec_on_host() {
     kubectl exec $(kubectl get pod -l app=debug -o jsonpath="{.items[0].metadata.name}") -- bash -c "nsenter -t 1 -m bash -c \"$1\"" > $2
 }
 
-create_storage_account() {
+create_storage_container() {
     set +x
-    az storage account create -n $WINDOWS_E2E_STORAGE_ACCOUNT -g $RESOURCE_GROUP_NAME -l $LOCATION
-    account_key=$(az storage account keys list --account-name $WINDOWS_E2E_STORAGE_ACCOUNT --query "[0].value")
-    az storage container create -n "tmp" --account-name $WINDOWS_E2E_STORAGE_ACCOUNT --account-key $account_key
+    # check if the storage container exists and create one if not
+    exists=$(az storage container exists --account-name $STORAGE_ACCOUNT_NAME --account-key $MAPPED_ACCOUNT_KEY --name $WINDOWS_E2E_STORAGE_CONTAINER)
+    if [[ $exists == *false* ]]; then
+        az storage container create -n $WINDOWS_E2E_STORAGE_CONTAINER --account-name $STORAGE_ACCOUNT_NAME --account-key $MAPPED_ACCOUNT_KEY
+        echo "Created storage container $WINDOWS_E2E_STORAGE_CONTAINER in $STORAGE_ACCOUNT_NAME"
+    else
+        # check if the storage container is empty and delete the blobs within one by one if not
+        blob_list=$(az storage blob list --account-name $STORAGE_ACCOUNT_NAME --container-name $WINDOWS_E2E_STORAGE_CONTAINER --account-key $MAPPED_ACCOUNT_KEY -o json | jq -r '.[] | .name')
```

Comment:
It seems that this part of code has not been tested in your shared pipeline links.
To test this part of code, I think you should select a kubernetes version which already has corresponding storage container, then delete the resource group and rerun the test pipeline to create new resource group.

Reply:
The RG of 1.24.9 has already been deleted. Requeued the pipeline and if you check the logs it says 

`Deleted blob aks-nodepool1-XXXXXXXX-vmss-linux-file.zip from storage container akswinstore2019-1249 in the storage account akswinstorageaccount`

Same goes for 1.26.3. 
`Deleted blob aks-nodepool1-XXXXXXXX-vmss-linux-file.zip from storage container akswinstore2019-1263 in the storage account akswinstorageaccount`